pages/Real-world usage.md

```diff
@@ -61,17 +61,24 @@ For example, we can start five Redix processes under our supervision tree and
 name them `:redix_0` to `:redix_4`:
 
 ```elixir
+# lib/my_app.ex
+
+# create the redix children list of workers
```

Comment:
Always start comments with a capital letter and end them with a `.` or a `:`; they're meant for humans :) Here I would say:

```elixir
# Create the list of Redix workers:
```

and call the variable above `redix_workers`. Wdyt?